packages/dev/core/src/scene.ts

```diff
@@ -779,7 +779,7 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
     /** Callback called when a pointer up is detected  */
     public onPointerUp: (evt: IPointerEvent, pickInfo: Nullable<PickingInfo>, type: PointerEventTypes) => void;
     /** Callback called when a pointer pick is detected */
-    public onPointerPick: (evt: IPointerEvent, pickInfo: PickingInfo) => void;
+    public onPointerPick?: (evt: IPointerEvent, pickInfo: PickingInfo) => void;

```

Comment:
All of these callbacks should technically have `?` since they all start as not defined. How much work is it to change all of them?